lib/usdUfe/ufe/UsdUndoDuplicateCommand.h

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright 2024 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef USDUFE_USDUNDODUPLICATECOMMAND_H
+#define USDUFE_USDUNDODUPLICATECOMMAND_H
+
+#include <usdUfe/base/api.h>
+#include <usdUfe/ufe/UsdSceneItem.h>
+#include <usdUfe/undo/UsdUndoableItem.h>
+
+#include <pxr/usd/sdf/path.h>
+
+#include <ufe/path.h>
+#include <ufe/undoableCommand.h>
+
+namespace USDUFE_NS_DEF {
+
+//! \brief UsdUndoDuplicateCommand
+class USDUFE_PUBLIC UsdUndoDuplicateCommand : public Ufe::SceneItemResultUndoableCommand
```

Comment:
First off, thanks for getting this command into MayaUSD.  Back when we started to implement the Clipboard in LookdevX, due to time constraints, we simply copied this command and modified the parts we were interested in. That's why we created a tech debt ticket, [EMSUSD-196](https://jira.autodesk.com/browse/EMSUSD-196), which aimed to 1. port the command into MayaUSD (and this PR did that), and 2. attempt to merge the differences with the original command. Additionally, I expect this command is gonna go through some more big changes since I think we wanna expand the scenarios it covers and how we do the duplicate.  Do you think we should create a new ticket (or just update [EMSUSD-196](https://jira.autodesk.com/browse/EMSUSD-196)) to try merging the differences, at least to keep track of this matter?

Reply:
Yes okay. I've taken note of that JIRA and will close it after I create a new one for the merging.